src/python.rs

```diff
@@ -58,10 +69,30 @@ impl Record {
 
 #[pymethods]
 impl Record {
+    #[getter]
+    pub fn name(&self) -> PyResult<&str> {
+        if let Some(pos) = self.id.find(char::is_whitespace) {
+            Ok(&self.id[..pos])
+        } else {
+            Ok(&self.id)
+        }
+    }
+
+    #[getter]
+    pub fn description(&self) -> PyResult<Option<&str>> {
+        if let Some(pos) = self.id.find(char::is_whitespace) {
+            Ok(Some(&self.id[pos..].trim_start()))
+        } else {
+            Ok(None)
+        }
+    }
+
+    #[getter]
     pub fn is_fasta(&self) -> PyResult<bool> {
```

Comment:
I agree that the getters are nicer but for reverse-compatibility's sake I think these should be left as-is.